examples/react-client/use-camera-and-microphone/src/MainControls.tsx

```diff
@@ -87,12 +87,10 @@ export const MainControls = () => {
   const disconnect = useDisconnect();
 
   const { localParticipant } = useParticipants();
-  const localTracks = localParticipant
-    ? [
-        ...localParticipant.cameraTracks,
-        ...localParticipant.screenshareVideoTracks,
-      ]
-    : [];
+  const localTracks = [
+    localParticipant?.cameraTrack,
+    localParticipant?.screenShareVideoTrack,
+  ];
```

Comment:
**suggestion:** instead of putting `?.fiter` in line 394, we could do sth like that here.
```suggestion
  const localTracks = [
    localParticipant?.cameraTrack,
    localParticipant?.screenShareVideoTrack,
  ].filter((track): track is Track => Boolean(track));
```